filemanager.lua

```diff
@@ -802,68 +799,87 @@ local function create_filedir(filedir_name, make_dir)
 		-- The scanlist is empty (or cursor is on ".."), so we add on our new file/dir at the bottom
 		scanlist[#scanlist + 1] = new_filedir
 		-- Add current position so it takes into account where we are
-		last_y = #scanlist + tree_view.Buf.Cursor.Loc.Y
+		last_y = #scanlist + tree_BufPane.Cursor.Loc.Y
 	end
 
 	refresh_view()
 	select_line(last_y)
 end
 
+-- parse_args will check if the passed in string array is empty
+-- @param args string array[]
+-- @return nil or string [nil if the string array is empty]
+function parse_args(args)
+	local return_string = nil
+	if #args == 0 then
+		-- args is empty
+	else
+	return_string = args[1]
+	end
+	return return_string
+end
```

Comment:
```suggestion
local function parse_args(args)
	if args == nil then
		micro.Log("parse_args given no args")
		return nil
	end
	local args_t = type(args)
	if args_t ~= "table" then
		if args_t == "string" then
			return args
		else
			micro.Log("parse_args not given a string or table type")
			return nil
		end
	end
	return #args > 0 and args[1] or nil
end
```

Reply:
I tried the above change and it passes back a userdata as type and it does not work. So left as is, as it is working and unless micro editor changes the api to the command parameter passing it will not need error checking.